src/algebra/algebra/operations.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kenny Lau
 -/
 import algebra.algebra.basic
+import linear_algebra.finsupp
```

Comment:
Is this needed? I don't see an obvious mention of finsupp below

Reply:
You're right: it is not (it was in a previous implementation and forgot it there), thanks.